index.ts

```diff
@@ -51,6 +52,26 @@ app.post(
     }
 )
 
+app.post(
+    "/searchItemName",
+    async (
+        req: Request<unknown, unknown, itemStuff, unknown>,
```

Comment:
Can you rename itemStuff to a better name please?

Reply:
Done.

Reply:
Done, chagned to itemInfo